SasModels Doc Build Failing

**Describe the bug**
Windows builds in the release_6.0.1 branch are failing with an error when building the sasmodels docs. See https://github.com/SasView/sasview/actions/runs/13290422326/job/37109590700 for more details.

Note - this might be an issue for sasmodels, but I haven't had a chance to dive into it.

**Expected behavior**
No error should be thrown and the doc build should succeed.

**SasView version (please complete the following information):**
 - Version: SasView release_6.0.1 branch which is building off of SasModel release_1.0.9 branch.

**Operating system (please complete the following information):**
 - OS: Windows CI (OSX and Ubuntu are successful)

**Additional context**
The error:

> C:\hostedtoolcache\windows\Python\3.11.9\x64\Lib\site-packages\pytools\persistent_dict.py:52: RecommendedHashNotFoundWarning: Unable to import recommended hash 'siphash24.siphash13', falling back to 'hashlib.sha256'. Run 'python3 -m pip install siphash24' to install the recommended hash.
>   warn("Unable to import recommended hash 'siphash24.siphash13', "
> C:\hostedtoolcache\windows\Python\3.11.9\x64\Lib\site-packages\pyopencl\cache.py:420: CompilerWarning: Non-empty compiler output encountered. Set the environment variable PYOPENCL_COMPILER_OUTPUT=1 to see more.
>   prg.build(options_bytes, [devices[i] for i in to_be_built_indices])
> Stack dump:
> 0.	Running pass 'CallGraph Pass Manager' on module 'main'.
> 0x00007FFD2BBF8887 (0x0000000000000000 0x0000000000000000 0x00000048729EADA0 0x0000022B83E46B00), RegisterGlobalAtExitNotification() + 0x1B7077 bytes(s)
> 0x00007FFD2BC68513 (0x0000022B83E46B00 0x00000048729EAD09 0x0000022B81693120 0x0000022B83E46AE0), RegisterGlobalAtExitNotification() + 0x226D03 bytes(s)
> 0x00007FFD2BC6839D (0x0000022BFAE262A0 0x0000022B816DE270 0x0000000000000080 0x00000048729EAE19), RegisterGlobalAtExitNotification() + 0x226B8D bytes(s)
> 0x00007FFD2BC68BB4 (0x0000000000000000 0x0000022BF5CF5F80 0x0000022BF5CF5F00 0x0000022B81693220), RegisterGlobalAtExitNotification() + 0x2273A4 bytes(s)
> 0x00007FFD2BA806DD (0x0000022B83FAD300 0x0000022BF6267770 0x0000022B0000000A 0x0000022BF6267A78), RegisterGlobalAtExitNotification() + 0x3EECD bytes(s)
> 0x00007FFD2BA80271 (0x0000022B816DD4F0 0x00000048729EB100 0x0000022BEE5B04D8 0x0000000000000000), RegisterGlobalAtExitNotification() + 0x3EA61 bytes(s)
> 0x00007FFD2B9595A1 (0x0000022B816DD4F0 0x00000048729EB0E0 0x00000048729EB100 0x00007FFD63DBA8C1), opencl_snprintf() + 0x7E1 bytes(s)
> 0x00007FFD2B941775 (0x0000022B8164EA60 0x0000022BFA2EA0D0 0x00000000000019A0 0x0000022B8164EA60)
> 0x00007FFD2B953790 (0x0000022BF5F4CC40 0x0000000000001AA9 0x0000022B81326B70 0x00007FFD618A646B)
> 0x00007FFD2B93F88C (0x00007FFD2C59F3CC 0x0000022BEE5B0470 0x0000022B81327F50 0x0000022BE5DC48B0)
> 0x00007FFD2C5486A2 (0x0000022BF5E374D8 0x0000022BFA844430 0x0000000000001AA9 0x0000022BF5E374D8), clDevInitDeviceAgent() + 0x2AB2 bytes(s)
> 0x00007FFD2C84CFB9 (0x0000022B00000014 0x0000022BEE5B0701 0x0000000000000000 0x0000022BFA844420), clWaitForEvents() + 0x459A9 bytes(s)
> 0x00007FFD2C738A4A (0x0000000000000014 0x0000022B80AA2050 0x0000022BEE5B0760 0x0000017E000001C0), ?TE_RegisterGlobalAtExitNotification@TaskExecutor@OpenCL@Intel@@YAXPEAVIAtExitCentralPoint@Utils@23@@Z() + 0x255A bytes(s)
> 0x00007FFD2C4D811E (0x0000022BEE7999F0 0x0000022BE57402B4 0x0000022BEE5B06C0 0x00007FFD63DBB57D), ?internal_execute@task_arena_base@internal@interface7@tbb@@IEBAXAEAVdelegate_base@234@@Z() + 0x6E bytes(s)
> 0x00007FFD2C725FF4 (0x0000022BE5740000 0x0000000000000000 0xFFFFFFFFFFFFFFFE 0x0000000000000000)
> 0x00007FFD2C80938D (0x00007FFD00000002 0x0000000000000000 0x0000022BFA844420 0x0000000000000000), clWaitForEvents() + 0x1D7D bytes(s)
> 0x00007FFD2C84DB12 (0x00000048729EB7E8 0x0000022B83E46458 0x0000000000000040 0x0000022B81673010), clWaitForEvents() + 0x46502 bytes(s)
> 0x00007FFD2C84CAC5 (0x0000022BF5E37428 0x0000000000000008 0x00007FFD2C8EAB98 0x0000022B83E46448), clWaitForEvents() + 0x454B5 bytes(s)
> 0x00007FFD2C86E0E1 (0x0000022BF5DFCEE0 0x0000022B83E46448 0x00007FFD2C8DFB28 0x0000000000000000), clWaitForEvents() + 0x66AD1 bytes(s)
> 0x00007FFD2C86DF03 (0x0000022B83E463E0 0x00000048729EB990 0x0000000000000000 0x00007FFD2C8E3C58), clWaitForEvents() + 0x668F3 bytes(s)
> 0x00007FFD2C86DE37 (0x00007FFD2C8EAE78 0x0000022B83E463E0 0x00007FFD2C8EAB98 0x0000022B83E46448), clWaitForEvents() + 0x66827 bytes(s)
> 0x00007FFD2C86E1A4 (0x0000022B83E41790 0x0000022B83E464F8 0x0000022B83E464F8 0x0000022B80AA2060), clWaitForEvents() + 0x66B94 bytes(s)
> 0x00007FFD2C84D3F3 (0x0000022B00000014 0x0000022BEE5B0701 0x0000022BEE7999F0 0x0000022BFA844420), clWaitForEvents() + 0x45DE3 bytes(s)
> 0x00007FFD2C738A4A (0x0000000000000014 0x0000022B815C99C0 0x0000022BEE5B0760 0x0000015000000000), ?TE_RegisterGlobalAtExitNotification@TaskExecutor@OpenCL@Intel@@YAXPEAVIAtExitCentralPoint@Utils@23@@Z() + 0x255A bytes(s)
> 0x00007FFD2C4D811E (0x0000022BEE7999F0 0x0000022BE57402AC 0x0000022BEE5B06C0 0x00007FFD63DBB57D), ?internal_execute@task_arena_base@internal@interface7@tbb@@IEBAXAEAVdelegate_base@234@@Z() + 0x6E bytes(s)
> 0x00007FFD2C725FF4 (0x0000022BE57F0000 0x0000000000000002 0xFFFFFFFFFFFFFFFE 0x0000000000000000)
> 0x00007FFD2C80938D (0x0000022B00000003 0x0000000000000002 0x0000022B815C9A90 0x0000022B815CB479), clWaitForEvents() + 0x1D7D bytes(s)
> 0x00007FFD2C84DB12 (0x00000048729EC488 0x0000022BFAB97A98 0x0000000000000040 0x0000022BF5F45020), clWaitForEvents() + 0x46502 bytes(s)
> 0x00007FFD2C84CAC5 (0x0000022B83E46448 0x0000000000000008 0x0000000000000000 0x0000000000000000), clWaitForEvents() + 0x454B5 bytes(s)
> 0x00007FFD2C86E0E1 (0x0000022BF5DFCD00 0x0000000000000001 0x0000022B815C99D0 0x00007FFD2C8A9557), clWaitForEvents() + 0x66AD1 bytes(s)
> 0x00007FFD2C86DF03 (0x0000022BFAB97A20 0x00000048729EC5C9 0x0000000000000000 0x00007FFD2C8E3C58), clWaitForEvents() + 0x668F3 bytes(s)
> 0x00007FFD2C86DE37 (0x00007FFD2C8EAD18 0x0000022BFAB97A20 0x0000000000000000 0x0000000000000000), clWaitForEvents() + 0x66827 bytes(s)
> 0x00007FFD2C86E1A4 (0x0000022B83E41CE0 0x0000022B815C99D0 0x0000022B815C99D0 0x0000022BFAB97B38), clWaitForEvents() + 0x66B94 bytes(s)
> 0x00007FFD2C84CDD1 (0x0000022BF9B9A130 0x0000022BF9D87E01 0x0000022BEE7999F0 0x0000000000000000), clWaitForEvents() + 0x457C1 bytes(s)
> 0x00007FFD2C738A4A (0x0000022BF9D87E00 0x0000000000000000 0x0000022BF9D87E00 0x0000022BF9D87E00), ?TE_RegisterGlobalAtExitNotification@TaskExecutor@OpenCL@Intel@@YAXPEAVIAtExitCentralPoint@Utils@23@@Z() + 0x255A bytes(s)
> 0x00007FFD2C4D7201 (0x0000000000000000 0x0000022BEE5B0760 0x00000048729EC779 0x0000022BF9D87E00), ?internal_current_slot@task_arena_base@internal@interface7@tbb@@KAHXZ() + 0x771 bytes(s)
> 0x00007FFD2C4D815C (0x0000022BEE7999F0 0x00007FFD2C8A53C9 0x0000022BEE5B06C0 0x00000048729EC8A8), ?internal_execute@task_arena_base@internal@interface7@tbb@@IEBAXAEAVdelegate_base@234@@Z() + 0xAC bytes(s)
> 0x00007FFD2C725FF4 (0x00000048729EC8A8 0x0000000000000010 0xFFFFFFFFFFFFFFFE 0x00007FFD2C80D930)
> 0x00007FFD2C80938D (0x00007FFD2C8DFB38 0x0000022BFAB98A70 0x0000022B00000001 0x000000[480](https://github.com/SasView/sasview/actions/runs/13290421431/job/37112247308?pr=2901#step:17:481)0000001), clWaitForEvents() + 0x1D7D bytes(s)
> 0x00007FFD2C84DB12 (0x0000000000000000 0x0000000000000000 0x0000000000000001 0x0000000000000000), clWaitForEvents() + 0x46502 bytes(s)
> 0x00007FFD2C84B9E8 (0x0000022BF5CEFB40 0x00000048729EDB78 0x0000022BFB6BF560 0x0000022BF5CEFB40), clWaitForEvents() + 0x443D8 bytes(s)
> 0x00007FFD2C83946D (0x0000000000000000 0x00000048729EDC78 0x0000022BFB6BF560 0x0000022BEC728E00), clWaitForEvents() + 0x31E5D bytes(s)
> 0x00007FFD2C82202F (0x0000000000000000 0x0000000000000000 0x00000048729EDD50 0x0000022BF92B0780), clWaitForEvents() + 0x1AA1F bytes(s)
> 0x00007FFD2C7CEF94 (0x00007FFD32217160 0x0000000000000001 0x0000022BEE89CCF0 0x00007FFD31D5E74E), clBuildProgram() + 0x6F4 bytes(s)
> 0x00007FFD35A9C3C2 (0x0000022BFC5D1CF0 0x0000022BF90AB040 0x000000[487](https://github.com/SasView/sasview/actions/runs/13290421431/job/37112247308?pr=2901#step:17:488)29EE350 0x0000022BF90AB040), clBuildProgram() + 0x42 bytes(s)
> 0x00007FFD2CA487D7 (0x00000048729EE3D0 0x0000022BF6F13401 0x00000048729EE318 0x00000048729EE3D0), PyInit__cl() + 0x2B2B7 bytes(s)
> 0x00007FFD2CA5E036 (0x0000000000000000 0x0000000000000000 0x0000022BF96D7DC0 0x0000022BFC3858B0), PyInit__cl() + 0x40B16 bytes(s)
> 0x00007FFD2CA5E0CC (0x00000048729EE658 0x00007FFD2CA7649E 0x00000048729EE658 0x00007FFD2CA7647D), PyInit__cl() + 0x40BAC bytes(s)
> 0x00007FFD2CA76A00 (0x0000022BF94F3E20 0x0000022BE57E1308 0x8000000000000001 0x0000022BF92324C0), ?what@python_error@nanobind@@UEBAPEBDXZ() + 0xB820 bytes(s)
> 0x00007FFD31CCC13B (0x0000000000000000 0x0000000000000000 0x00000048729EE750 0x0000022B00000001), PyObject_Vectorcall() + 0x5EB bytes(s)
> 0x00007FFD31CCD423 (0x00007FFD3223F960 0x0000022BEED1BF60 0x0000000000000000 0x00000048729EE990), _PyEval_EvalFrameDefault() + 0x803 bytes(s)
> 0x00007FFD31CCAE64 (0x0000022BE57E07B8 0x00000048729EE990 0x0000022BE57E07A8 0x0000000000000000), _PyFunction_Vectorcall() + 0x1A4 bytes(s)
> 0x00007FFD31CA7DA3 (0x00000048729EE990 0x0000000000000005 0x00000000A64439C9 0x00007FFD31D1E1C5), _PyObject_FastCallDictTstate() + 0x57 bytes(s)
> 0x00007FFD31CA625B (0x0000022B8000F970 0x00007FFD3223F960 0x0000022BEED1BF60 0x00007FFD320AED18), _PyObject_Call_Prepend() + 0x7F bytes(s)
> 0x00007FFD31CA6188 (0x0000022B8000F970 0x0000000000000000 0x0000000000000000 0x00007FFD00000001), _Py_parse_inf_or_nan() + 0x52C bytes(s)
> 0x00007FFD31CC83DE (0x8000000000000004 0x0000000000000000 0x0000022BE57E0770 0x0000022BEE56CA6A), _PyObject_MakeTpCall() + 0x37E bytes(s)
> 0x00007FFD31CCBDDF (0x0000022BF92B0780 0x00007FFD3223F960 0x0000000000000003 0x00000000000003B3), PyObject_Vectorcall() + 0x28F bytes(s)
> 0x00007FFD31CCD423 (0x00007FFD3223F960 0x0000022BF5171D00 0x000make: *** [Makefile:55: ref/models/index.rst] Error 2816

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -179,6 +179,8 @@ jobs:
 
       - name: Build sasmodels, sasdata, and bumps docs
         if: ${{ matrix.docs }}
+        env:
+          SAS_OPENCL: NONE
         run: |
           make -C ../bumps/doc html || true
           mkdir -p ~/.sasmodels/compiled_models

--- a/build_tools/requirements.txt
+++ b/build_tools/requirements.txt
@@ -24,6 +24,7 @@ pytest_qt
 pytest-mock
 pytools
 qtconsole
+siphash24
 six
 sphinx
 superqt

